src/core/libraries/companion/companion_util.cpp

```diff
@@ -0,0 +1,83 @@
+// SPDX-FileCopyrightText: Copyright 2025 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "common/logging/log.h"
+#include "core/libraries/error_codes.h"
+#include "core/libraries/libs.h"
+#include "core/libraries/companion/companion_util.h"
+
+namespace Libraries::CompanionUtil {
+
+uint32_t PS4_SYSV_ABI getEvent(sceCompanionUtilContext* ctx, sceCompanionUtilEvent* outEvent,
```

Comment:
```suggestion
u32 PS4_SYSV_ABI getEvent(sceCompanionUtilContext* ctx, sceCompanionUtilEvent* outEvent,
```
You can use the typedefs of the project for these, and same for all the other occurrences